parsing/g_constr.mlg

```diff
@@ -20,9 +20,9 @@ open Constrexpr
 open Constrexpr_ops
 open Util
 
-open Pcoq
-open Pcoq.Prim
-open Pcoq.Constr
+open Procq
+open Procq.Prim
+open Procq.Constr
```

Comment:
Not to get in the way of progress, but isn't this mighty likely to break plugins that parse things?  Do we care?  Would be nice for the release notes to mention some of the things that will break when this is committed.

Reply:
Indeed. We have a `dev/doc/changes.md` file which should be used for this kind of notes.

Reply:
The alias will keep things working, at the cost of a deprecation warning. I did check on reasonable examples.